data.sql

```diff
@@ -10,4 +10,27 @@ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_
 VALUES (3, 'Pikachu', DATE '2021-01-07', 1, false, 15.04);
 
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) 
-VALUES (4, 'Devimon', DATE '2017-05-12', 5, true, 11.0);
\ No newline at end of file
+VALUES (4, 'Devimon', DATE '2017-05-12', 5, true, 11.0);
```

Comment:
- [ ] That line is showing as being added within this pull request, but as you said, make sure you just remove it to avoid confusion to other code reviews

Reply:
Honestly, I don’t understand what you mean because that has to do with GitHub and how changes are shown to the user.

Reply:
@joskalenda Thanks for your time 🙏🏼.